x-pack/plugins/beats_management/public/lib/adapters/tags/adapter_types.ts

```diff
@@ -0,0 +1,11 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import { BeatTag } from '../../../../common/domain_types';
+
+export interface CMTagsAdapter {
+  getTagsWithIds(tagIds: string[]): Promise<BeatTag[]>;
+  upsertTag(tag: BeatTag): Promise<{}>;
```

Comment:
Just a general TypeScript question, what's the difference between returning `Promise<{}>` and something like `Promise<any>`?